numpy/ma/core.py

```diff
@@ -2085,6 +2085,8 @@ def masked_equal(x, value, copy=True):
     `condition` = (x == value).  For floating point arrays,
     consider using ``masked_values(x, value)``.
 
+    The fill_value is set to `value`.
```

Comment:
This isolated paragraph/sentence doesn't make sense. Could you try rewriting the whole thing together with the previous paragraph?

Reply:
Hope it is more explicit now.